config/environments/development.rb

```diff
@@ -58,4 +58,9 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :test
+  host = 'localhost:3000'
+  config.action_mailer.default_url_options = { host: host, protocol: 'http' }
```

Comment:
```suggestion
  config.action_mailer.default_url_options = {host: host, protocol: "http"}
```
dùng nháy kép em nhé, bỏ khoảng trắng sau `{` và trước `}`